tests/test.py

```diff
@@ -11,7 +11,31 @@
 import sys
 
 from pathlib import Path
-from typing import Callable
+from typing import Callable, TypeAlias, TypedDict, Union
+
+class Device(TypedDict):
+    major: int
+    minor: int
+class Parent(TypedDict, total=False):
```

Comment:
Hmm... Why do we have ruff when it doesn't complain about missing newlines between class definitions? I want my money back! :wink: